### PR TITLE
Update index.md to include AutoMLPipeline

### PR DESCRIPTION
**Pull request by ppalmes**

### CODE REVIEW

**Comment by bkamins** [comment]
Thank you for the contribution. Let us just wait for a language review before merging.

File: docs/src/index.md
Comment (nalimilan) [inline docs/src/index.md:71]:
```suggestion
      A package that makes it trivial to create complex ML 
```

**Review by nalimilan — COMMENTED** [review]
I have (somewhat surprisingly :-) very few suggestions to make.

**Review by bkamins — APPROVED** [review]

**Comment by ppalmes** [comment]
thanks all!

**Pull request merged**
